packages/expo/src/useOAuth.ts

```diff
@@ -0,0 +1,84 @@
+import { useSignIn, useSignUp } from '@clerk/clerk-react';
+import type { OAuthStrategy } from '@clerk/types';
+import * as AuthSession from 'expo-auth-session';
+import * as WebBrowser from 'expo-web-browser';
+
+export type UseOAuthFlowParams =
```

Comment:
❓ Is there a specific reason we're changing the signature of the function instead of skipping redirectUrl?

```tsx
useOAuth({ strategy: 'oauth_google' })
```
vs
```tsx
useOAuth('oauth_google')
```

I think we usually go with the 1rst one as its more extensible.